sw/airborne/modules/sensors/temp_adc.h

```diff
@@ -28,12 +28,6 @@
 
 #include "std.h"
 
-/// flag to enable sending every new measurement via telemetry
-bool temp_adc_sync_send;
```

Comment:
Because in my opinion this was only added for debugging and instead I made the define for that.